projects/control-service/projects/pipelines_control_service/src/main/resources/db/migration/V20231130230000__rename_column_vdk_version_to_vdk_image_in_datajob_executions.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists data_job_execution
+    rename column vdk_version to vdk_image;
```

Comment:
We should not change the data_job_execution table since the value there comes from the pod termination message.

Reply:
Removed, thanks